### ISSUE TITLE
Proper way to pass more than one positional argument

### ISSUE THREAD

**Issue by annisat** [clokep/celery-batches#38]
Hi,

I'm using celery-batches and following the example in the documentation, with modification to fit my own need.
I'm expecting to use the decorated function with `get_batch_result.delay(char, font)`. However, I don't know how to set the function's argument accordingly.
I tried `def get_batch_result(requests)`, following error occurs when the task came in:
```
TypeError: get_batch_result() takes 1 positional argument but 2 were given
```
If I try `def get_batch_result(requests1, requests2)` , task came in and it prints:
```
TypeError: get_batch_result() missing 1 required positional argument: 'requests2'
```
I also tried `def get_batch_result(*requests)`. Now requests is a tuple, containing one list of SimpleRequest. The tuple part doesn't seem to follow what the document describes. I'm not sure if I'm doing this right.
Is there a standard way to do this? Or currently I should pack my argument as a list or tuple or kwargs as a work-around.

**Comment by annisat** [comment]
Yes of course. My environment is based on docker image `python:3.8`
Service-related packages:
```
Flask==2.0.2
celery==5.2.2
celery-batches==0.6
redis==4.1.0
eventlet==0.33.0
```
Job-related packages:
```
numpy==1.19.5
torch==1.9.1
pillow==8.3.2
opencv-python==4.5.3.56
```
Docker command is a `run.sh` and the celery-related line is `celery -A worker worker --loglevel=INFO --concurrency=500 --pool=eventlet &`

**Comment by clokep** [comment]
A better workaround is to set `typing=False` on the task itself:

```python
@worker.task(base=Batches, flush_every=16, flush_interval=0.5, typing=False)
def get_batch_result(*requests):
    ....
```

The `BatchTask` should probably do this automatically as the typing will almost never line up right now.

**Comment by clokep** [comment]
I believe #39 will fix this for most use-cases.

**Comment by clokep** [comment]
I remember why I never ran into this -- I tend to run batch tasks using `app.send_task(...)` instead of `task.delay(...)`, `send_task` just puts the task name in manually and skips a lot of the validation.

I've fixed this in #39 though and it should be in the next release. 👍 Thanks for the bug report!
